clifford/_utils.py

```diff
@@ -0,0 +1,43 @@
+""" Hacks for numba """
+import functools
+import numba
+
+
+class pickleable_function:
+    def __init__(self, func):
+        self.__func = func
+        functools.update_wrapper(self, self.__func)
+
+    @classmethod
+    def _rebuild(cls, *args):
+        return cls(numba.serialize._rebuild_function(*args))
+
+    def __reduce__(self):
+        globs = numba.serialize._get_function_globals_for_reduction(self.__func)
+        return (
+            self._rebuild,
+            numba.serialize._reduce_function(self.__func, globs)
+        )
+
+    def __call__(self, *args, **kwargs):
+        return self.__func(*args, **kwargs)
+
+    def __repr__(self):
+        return "_pickleable_function({!r})".format(self.__func)
```

Comment:
Tried upstreaming this in https://github.com/numba/numba/pull/5340

@stuartarchibald, do you think something like this would be an acceptable solution to https://github.com/numba/numba/issues/4759?